### PR TITLE
Localization: Localized game-data entries

### PR DESCRIPTION
**Pull request by josericardo-fo**
## What are the changes?

Localization and translations
Some translation don't fit and other ones had to ommit information in order to fit. A suggestion would be to make this box larger or make the font smaller.

| Language |                                                   Preview                                                  |
|:--------:|:----------------------------------------------------------------------------------------------------------:|
|    de    | image |
|    fr    | image |
|    ko    | image |
|   pt_br  | image |

## Checklist
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?
- [X] Have I tested the changes (manually)?
    - [X] Are all unit tests still passing? (`npm run test`)
- [X] Are the changes visual?
  - [X] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/fr/game-data.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "systemData": "system",
+  "sessionData": "session",
+  "settingsData": "settings",
+  "tutorialsData": "tutorials",
+  "seenDialoguesData": "seen dialogues",
+
+  "dataCouldNotBeLoaded": "Your {{dataName}} data could not be loaded. It may be corrupted.",
+  "dataWillBeOverridden": "Your {{dataName}} data will be overridden and the page will reload. Proceed?",
+  "errorContactServer": "Could not contact the server. Your {{dataName}} data could not be imported.",
+  "errorUpdating": "An error occurred while updating {{dataName}} data. Please contact the administrator.",
```
Comment (Adri1) [inline src/locales/fr/game-data.ts:13]:
```suggestion
  "systemData": "système",
  "sessionData": "de session",
  "settingsData": "des paramètres",
  "tutorialsData": "des tutoriels",
  "seenDialoguesData": "des dialogues connus",

  "dataCouldNotBeLoaded": "Impossible de charger les données {{dataName}}. Elles sont peut-être corrompues.",
  "dataWillBeOverridden": "Les données {{dataName}} seront effacées et la page rafraichie. Continuer ?",
  "errorContactServer": "Connexion au serveur impossible. Les données {{dataName}} n’ont pas eu être importées.",
  "errorUpdating": "Une erreur est survenue pendant le chargement des données {{dataName}}. Contactez un administrateur.",
```

**Review by Adri1 — COMMENTED** [review]
Added French

File: src/locales/fr/game-data.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "systemData": "système",
+  "sessionData": "de session",
+  "settingsData": "des paramètres",
+  "tutorialsData": "des tutoriels",
+  "seenDialoguesData": "des dialogues connus",
+
+  "dataCouldNotBeLoaded": "Impossible de charger les données {{dataName}}. Elles sont peut-être corrompues.",
+  "dataWillBeOverridden": "Les données {{dataName}} seront effacées et la page rafraichie. Continuer ?",
+  "errorContactServer": "Connexion au serveur impossible. Les données {{dataName}} n’ont pas eu être importées.",
+  "errorUpdating": "Une erreur est survenue pendant le chargement des données {{dataName}}. Contactez un administrateur.",
```
Comment (Adri1) [inline src/locales/fr/game-data.ts:13]:
```suggestion
  "dataCouldNotBeLoaded": "Impossible de charger les données {{dataName}}. Elles sont peut-être corrompues.",
  "dataWillBeOverridden": "Les données {{dataName}} seront effacées. Continuer ?",
  "errorContactServer": "Connexion au serveur impossible. Les données {{dataName}} n’ont pas eu être importées.",
  "errorUpdating": "Une erreur est survenue lors du chargement des données {{dataName}}. Contactez un administrateur.",
```

**Review by Adri1 — COMMENTED** [review]
French corrections

File: src/locales/de/game-data.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "systemData": "system",
+  "sessionData": "session",
+  "settingsData": "settings",
+  "tutorialsData": "tutorials",
+  "seenDialoguesData": "seen dialogues",
+
+  "dataCouldNotBeLoaded": "Your {{dataName}} data could not be loaded. It may be corrupted.",
+  "dataWillBeOverridden": "Your {{dataName}} data will be overridden and the page will reload. Proceed?",
+  "errorContactServer": "Could not contact the server. Your {{dataName}} data could not be imported.",
+  "errorUpdating": "An error occurred while updating {{dataName}} data. Please contact the administrator.",
```
Comment (CodeTappert) [inline src/locales/de/game-data.ts:13]:
```suggestion
  "systemData": "System",
  "sessionData": "Session",
  "settingsData": "Einstellungen",
  "tutorialsData": "Tutorials",
  "seenDialoguesData": "Gesehener Dialog",

  "dataCouldNotBeLoaded": "Deine {{dataName}} Daten konnten nicht geladen werden. Möglicherweise sind sie beschädigt.",
  "dataWillBeOverridden": "Deine {{dataName}} Daten werden überschrieben und die Seite wird neugeladen. Fortfahren?",
  "errorContactServer": "Konnte den Server nicht kontaktieren. Deine {{dataName}} Daten konnten nicht importiert werden.",
  "errorUpdating": "Es ist ein Fehler aufgetreten während die {{dataName}} aktualisiert wurden. Kontaktiere bitte einen Administrator.",
```

**Review by CodeTappert — APPROVED** [review]

**Review by Adri1 — APPROVED** [review]

File: src/locales/ko/game-data.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  "systemData": "system",
+  "sessionData": "session",
+  "settingsData": "settings",
+  "tutorialsData": "tutorials",
+  "seenDialoguesData": "seen dialogues",
+
+  "dataCouldNotBeLoaded": "Your {{dataName}} data could not be loaded. It may be corrupted.",
+  "dataWillBeOverridden": "Your {{dataName}} data will be overridden and the page will reload. Proceed?",
+  "errorContactServer": "Could not contact the server. Your {{dataName}} data could not be imported.",
+  "errorUpdating": "An error occurred while updating {{dataName}} data. Please contact the administrator.",
```
Comment (EnochG1) [inline src/locales/ko/game-data.ts:13]:
```suggestion
  "systemData": "시스템",
  "sessionData": "세션",
  "settingsData": "설정",
  "tutorialsData": "튜토리얼",
  "seenDialoguesData": "본 대화",

  "dataCouldNotBeLoaded": "{{dataName}}[[를]] 불러올 수 없었습니다. 파일이 손상되었을 수 있습니다.",
  "dataWillBeOverridden": "{{dataName}} 데이터를 덮어쓰고 페이지를 새로고침 합니다. 계속하시겠습니까?",
  "errorContactServer": "서버에 접근할 수 없습니다. {{dataName}} 데이터를 불러올 수 없습니다.",
  "errorUpdating": "{{dataName}} 데이터를 업데이트하는 도중 오류가 발생했습니다. 관리자에게 문의하십시오.",
```

**Review by EnochG1 — COMMENTED** [review]
Korean Translation

File: src/locales/de/game-data.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+export const gameData: SimpleTranslationEntries = {
+  "systemData": "System",
+  "sessionData": "Session",
+  "settingsData": "Einstellungen",
+  "tutorialsData": "Tutorials",
+  "seenDialoguesData": "Gesehener Dialog",
+
+  "dataCouldNotBeLoaded": "Deine {{dataName}} Daten konnten nicht geladen werden. Möglicherweise sind sie beschädigt.",
+  "dataWillBeOverridden": "Deine {{dataName}} Daten werden überschrieben und die Seite wird neugeladen. Fortfahren?",
```
Comment (CodeTappert) [inline src/locales/de/game-data.ts:11]:
```suggestion
  "dataWillBeOverridden": "Deine {{dataName}} Daten werden über-/nschrieben und es wird neugeladen. Fortfahren?",
```

**Review by EnochG1 — APPROVED** [review]

File: src/locales/es/game-data.ts [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+  "systemData": "system",
+  "sessionData": "session",
+  "settingsData": "settings",
+  "tutorialsData": "tutorials",
+  "seenDialoguesData": "seen dialogues",
+
+  "dataCouldNotBeLoaded": "Your {{dataName}} data could not be loaded. It may be corrupted.",
+  "dataWillBeOverridden": "Your {{dataName}} data will be overridden and the page will reload. Proceed?",
+  "errorContactServer": "Could not contact the server. Your {{dataName}} data could not be imported.",
+  "errorUpdating": "An error occurred while updating {{dataName}} data. Please contact the administrator.",
```
Comment (InnocentGameDev) [inline src/locales/es/game-data.ts:13]:
```suggestion
  "systemData": "sistema",
  "sessionData": "sesión",
  "settingsData": "ajustes",
  "tutorialsData": "tutoriales",
  "seenDialoguesData": "diálogos leídos",

  "dataCouldNotBeLoaded": "Tus datos "{{dataName}}" no se han podido cargar. Puede que estén corruptos.",
  "dataWillBeOverridden": "Tus datos "{{dataName}}" se sobreescribirán y la página recargará, ¿estás seguro?",
  "errorContactServer": "No se pudo contactar con el servidor. Tus datos "{{dataName}}" no han podido ser importados.",
  "errorUpdating": "Un error ha ocurrido mientras se subían los datos "{{dataName}}". Por favor, ponte en contacto con el administrador.",
```

**Review by InnocentGameDev — CHANGES_REQUESTED** [review]
Added "es" localization

File: src/locales/es/game-data.ts [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+  "systemData": "sistema",
+  "sessionData": "sesión",
+  "settingsData": "ajustes",
+  "tutorialsData": "tutoriales",
+  "seenDialoguesData": "diálogos leídos",
+
+  "dataCouldNotBeLoaded": "Tus datos "{{dataName}}" no se han podido cargar. Puede que estén corruptos.",
+  "dataWillBeOverridden": "Tus datos "{{dataName}}" se sobreescribirán y la página recargará, ¿estás seguro?",
+  "errorContactServer": "No se pudo contactar con el servidor. Tus datos "{{dataName}}" no han podido ser importados.",
+  "errorUpdating": "Un error ha ocurrido mientras se subían los datos "{{dataName}}". Por favor, ponte en contacto con el administrador.",
```
Comment (InnocentGameDev) [inline src/locales/es/game-data.ts:13]:
```suggestion
  "dataCouldNotBeLoaded": "Tus datos {{dataName}} no se han podido cargar. Puede que estén corruptos.",
  "dataWillBeOverridden": "Tus datos {{dataName}} se sobreescribirán y la página recargará, ¿estás seguro?",
  "errorContactServer": "No se pudo contactar con el servidor. Tus datos {{dataName}} no han podido ser importados.",
  "errorUpdating": "Un error ha ocurrido mientras se subían los datos {{dataName}}. Por favor, ponte en contacto con el administrador.",
```

**Comment by josericardo-fo** [comment]
Closed until future changes

**Pull request closed without merging**